virtool/account/api.py

```diff
@@ -92,39 +93,22 @@ async def patch(
             400: Invalid input
             401: Requires Authorization
         """
-        db = self.request.app["db"]
-        user_id = self.request["client"].user_id
-
-        update = {}
-
-        data = data.dict(exclude_unset=True)
+        password = data.password
 
-        if "password" in data:
-            # Request model ensures that if one password is passed in,
-            # the other is as well.
-            if error := await check_password_length(self.request, data["password"]):
+        if password is not None:
```

Comment:
Model should not be using default values for a PATCH endpoint.

Reply:
so that means remove the Optional keyword right? And a request body containing no fields for a PATCH request should end in a status code of 400?

Reply:
No.

Remember we are using `exclude_unset` to drop fields that are not set in the request body. The fields are optional, but they do not have default values.

A `400` is automatically returned by `aiohttp-pydantic` when the request JSON is `'{}'`. That is not valid JSON—it must have at least one key.